### ISSUE TITLE
matemat@matemat.hq.c3d2.de ist hart in Handler/Common.hs eincompiliert

### ISSUE THREAD

**Issue by Florz** [nek0/yammat#125]
Die Zeile
`("List-Id", "\"Matemat\" <matemat@matemat.hq.c3d2.de>")]`
in `Handler/Common.hs`  sollte in die Konfigurationsdatei ausgelagert werden.

**Comment by nek0** [comment]
Stimmt. Brauchst du die Änderung unbedingt, denn eigentlich arbeite ich schon am Nachfolger.

**Comment by Florz** [comment]
Nö, ist nicht so dringend. Wir haben halt yammat gerade produktiv genommen bei uns. Aber wir steigen auch gerne um, wenns was neues gibt.
